lib/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -88,3 +88,38 @@ a { color: pink }
 ```css
 a { background: orange; color: pink }
 ```
+
+## Optional secondary options
+
+### `ignore: ["single-declaration"]`
+
+#### `"single-declaration"`
+
+The following patterns are _not_ considered violations:
+
+<!-- prettier-ignore -->
+```css
+foo { property: value }
+```
+
+<!-- prettier-ignore -->
+```css
+foo { property: value; }
+```
+
+<!-- prettier-ignore -->
+```css
+@keyframes name { from { property: 0 } to { property: 1; } }
+```
+
+With `"always"`:
+
+The following pattern is _still_ considered a violation:
+
+<!-- prettier-ignore -->
+```css
+foo {
+  property: value;
+  bar: qux
+}
+```
```

Comment:
```suggestion
```


Let's remove this example to be consistent with other rules where we only show the rule being more permissive in the examples and, due to a limitation in how the website is built, the example won't be given the correct background colour.